### PR TITLE
Adds elf ears for humans.

### PR DESCRIPTION
**Pull request by Higgin**
## About The Pull Request

Adds a small cosmetic genemod for humans - elf ears.

## Why It's Good For The Game

Lore-friendly customization options... good? This also includes a new define to make future recolorable sprite accessories for humans mirror their skin-tones, code help thanks to @MarkSuckerberg 

Original ears from [Skyrat](https://github.com/Skyrat-SS13/Skyrat-tg/pull/675).

## Changelog

:cl:
add: Adds elf ears.
/:cl:

image
image
image

### CODE REVIEW

File: code/modules/surgery/organs/ears.dm [hunk 1]
```diff
@@ -166,6 +166,29 @@
...
+/obj/item/organ/ears/elf
+	name = "elf ears"
+	icon = 'icons/obj/clothing/hats.dmi'
+	icon_state = "elf"
+	damage_multiplier = 1.5
+
+/obj/item/organ/ears/elf/Insert(mob/living/carbon/human/ear_owner, special = 0, drop_if_replaced = TRUE)
+	..()
+	if(istype(ear_owner))
+		color = ear_owner.skin_tone
```
Comment (MarkSuckerberg) [inline code/modules/surgery/organs/ears.dm]:
```suggestion
		color = skintone2hex(ear_owner.skin_tone)
```

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]
just this, pretty much!

**Comment by thgvr** [comment]
Any chance you could make these use the same greyscale pallette as human parts?

**Review by MarkSuckerberg — APPROVED** [review]
code is good, though I do think it would be nice if you could re-palette the ears. Shouldn't be too difficult!

**Comment by Higgin** [comment]
image
image
image
@thgvr @MarkSuckerberg applied the greyscale human palette, how's this look?

**Review by thgvr — APPROVED** [review]

**Pull request merged**